enacts/wat_bal/enactstozarr.py

```diff
@@ -0,0 +1,59 @@
+import os
```

Comment:
Sounds good. I only copied from onset/ to "stick the zarrification to the app" so that we know where the data is coming from. Shall I move onset/enactstozarr.py one level down and merge it with the one in monhtly/ ?

Reply:
If you meant "one level up" instead of down, then yes. Otherwise I don't think I understand what you're suggesting.